spec/operator_spec.lua

```diff
@@ -34,7 +34,38 @@ describe('operator', function()
 				{a = 5, b = 'cedf'},
 				{a = 4, b = 'sfd'},
 			}
+			local bar = {
+				{a = {b = {c = 'd1'}, e = 'f1', 's1'}, g = 'h1', 't1', ['5'] = 'q1'},
+				{a = {b = {c = 'd2'}, e = 'f2', 's2'}, g = 'h2', 't2', ['5'] = 'q2'},
+			}
 			assert.are_same({3, 5, 4}, Array.map(foo, Operator.property('a')))
+			assert.are_same({{c = 'd1'}, {c = 'd2'}}, Array.map(foo, Operator.property('a.b')))
+			assert.are_same({'d1', 'd2'}, Array.map(foo, Operator.property('a.b.c')))
+			assert.are_same({'f1', 'f2'}, Array.map(foo, Operator.property('a.e')))
+			assert.are_same({'s1', 's2'}, Array.map(foo, Operator.property('a.1')))
+			assert.are_same({'t1', 't2'}, Array.map(foo, Operator.property(1)))
+			assert.are_same({'t1', 't2'}, Array.map(foo, Operator.property('1')))
+			assert.are_same({'q1', 'q2'}, Array.map(foo, Operator.property('5')))
+			assert.are_same({'q1', 'q2'}, Array.map(foo, Operator.property(5)))
+			assert.is_nil( Operator.property(5)(foo))
+			assert.error(Operator.property)
+			assert.error(Operator.property('a'))
```

Comment:
Assert.error takes a function name as input

Reply:
`Operator.property('a')` is a function awaiting a table as input, instead it gets no input at all, hence errors, this is the intended test case here